content/en/docs/setup/install/standalone-operator/index.md

```diff
@@ -171,6 +171,56 @@ $ kubectl logs -f -n istio-operator $(kubectl get pods -n istio-operator -lname=
 Refer to the [`IstioOperator` API](/docs/reference/config/istio.operator.v1alpha1/#IstioOperatorSpec)
 for the complete set of configuration settings.
 
+## Canary Upgrade
+
+You can use operator to do canary upgrade of Istio control plane, similar to what istioctl does, refer to [upgrade guide]((/docs/setup/upgrade/#canary-upgrades))
+
+For example, if you have already installed Istio following the previous step, check existence of in-cluster `IstioOperator` CR "example-istiocontrolplane":
+
+{{< text bash >}}
+$ kubectl get iop --all-namespaces
+NAMESPACE      NAME                        REVISION   STATUS    AGE
+istio-system   example-istiocontrolplane              HEALTHY   11m
+{{< /text >}}
+
+Then run the following command with corresponding version of istioctl to install Istio of new revision based on the in-cluster `IstioOperator` CR:
+
+{{< text bash >}}
+$ istioctl operator init --revision canary
```

Comment:
don't use canary, we're moving away from that. instead use an actual revision label like istio-1-6-7

Reply:
done